Vagrantfile

```diff
@@ -67,7 +67,17 @@ Vagrant.configure("2") do |config|
 	if CONF['ip'] == "dhcp"
 		config.vm.network :private_network, type: "dhcp", hostsupdater: "skip"
 	else
-		config.vm.network :private_network, ip: CONF['ip'], hostsupdater: "skip"
+		config.vm.network :private_network, ip: CONF['ip']
+		# Add IP addresses of the other domains to /etc/hosts
+		if defined?(VagrantPlugins::HostsUpdater)
```

Comment:
I added this as Avahi wasn't working for me 😢

I think I can clean up the logic here and not use `.clone` if we go with this.